content/en/providers/cloud-compute/openstack/aai.md

```diff
@@ -737,19 +740,29 @@ Similarly to the change from Demo to Production described
 [above](#moving-to-egi-check-in-production-instance), you will need to update
 the URLs on your configuration:
 
-1. Update the `remote-id` of the identity provider:
+1. Update the `remote-id` of the identity provider to also include the new
+   issuer:
 
    ```shell
-   $ openstack identity provider set --remote-id https://aai.egi.eu/auth/realms/egi egi.eu
+   $ openstack identity provider set --remote-id https://aai-dev.egi.eu/auth/realms/egi --remote-id https://aai-dev.egi.eu/oidc/ egi.eu
```

Comment:
I suppose that the instruction refer to the production Check-in , so the command should be:

```
openstack identity provider set --remote-id https://aai.egi.eu/auth/realms/egi --remote-id https://aai.egi.eu/oidc/ egi.eu
```

Reply:
yup, copy pasted too much